python/ray/experimental/data/tests/test_dataset.py

```diff
@@ -623,8 +625,7 @@ def test_csv_read(ray_start_regular_shared, tmp_path):
     assert df1.equals(dsdf)
     # Test metadata ops.
     assert ds.count() == 3
-    assert "two" in str(ds.schema())
-    assert "two" in str(ds)
+    assert ds.input_files() == [path1]
```

Comment:
Add back.

Reply:
I thought that we're punting on schema fixup for now?